### PR TITLE
WALL-43: Create Input comp

### PR DESCRIPTION
**Pull request by manikangkan**
## Summary
Create Input comp

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code change that neither fixes a bug nor adds a feature)

### Linked Issues

Null

## JIRA Issue(s)

- [JIRA Issue Link](https://wallaceproject.atlassian.net/browse/WALL-43)

## Checklist (check all)

- [x] Automated tests have been added, or an explanation for why not
- [ ] Security implications and dependencies on linked features have been considered and explained

## Screenshots & Evidence

_Include any relevant screenshots, Videos, or links to documentation that provide additional context or proof of functionality._
image

image

### CODE REVIEW

File: src/components/atoms/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+            {inputTitle}
+          </label>
+        )}
+        <div className="relative">
+          {leftIcon && (
+            <div className="absolute inset-y-0 left-0 flex items-center pl-[12px]">{leftIcon}</div>
+          )}
+          {showCurrencySymbol && (
+            <div className="absolute inset-y-0 left-0 flex items-center pl-[20px]">$</div>
+          )}
```
Comment (raj-edition) [inline src/components/atoms/Input/Input.tsx:122]: ```leftIcon || showCurrencySymbol && <div>{leftIcon || '$'}</div>```

File: src/components/atoms/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          {rightIcon && (
+            <div className="absolute inset-y-0 right-0 flex items-center pr-[12px]">
+              {rightIcon}
+            </div>
+          )}
+          {rightText && (
+            <div className="absolute inset-y-0 right-0 flex items-center pr-[12px] text-sm text-gray-400">
+              {rightText}
+            </div>
+          )}
```
Comment (raj-edition) [inline src/components/atoms/Input/Input.tsx:146]:
```rightIcon || RightText &&
<div>
{leftIcon || '$'}
</div>
```

**Review by raj-edition — CHANGES_REQUESTED** [review]
One change request

File: src/components/atoms/Input/Input.tsx (hunk 1 above)
Comment (manikangkan) [reply src/components/atoms/Input/Input.tsx]: Sorted @raj-edition

File: src/components/atoms/Input/Input.tsx (hunk 2 above)
Comment (manikangkan) [reply src/components/atoms/Input/Input.tsx]: Sorted @raj-edition

File: src/components/atoms/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            <div className="absolute inset-y-0 right-0 flex items-center pr-[12px]">
+              {rightIcon && <span className="mr-2">{rightIcon}</span>}
+              {rightText && (
+                <span className="text-[16px] leading-[160%] tracking-[0%] font-[400] text-black/80">
+                  {rightText}
+                </span>
+              )}
+            </div>
+          )}
+        </div>
```
Comment (raj-edition) [inline src/components/atoms/Input/Input.tsx:153]: is there a scenario where both righticon and right text will be visible @manikangkan ?

Comment (manikangkan) [reply src/components/atoms/Input/Input.tsx:153]: No @raj-edition

Comment (raj-edition) [reply src/components/atoms/Input/Input.tsx:153]:
```
 {(rightIcon || rightText) && (
            <div className="absolute inset-y-0 right-0 flex items-center pr-[12px]">
               <span className={rightIcon? "mr-2": "text-[16px] leading-[160%] tracking-[0%] font-[400] text-black/80"}>{rightIcon?? rightText}</span>
            </div>
          )}
```

**Review by raj-edition — APPROVED** [review]

**Pull request merged**